types/src/auction/providers.rs

```diff
@@ -9,13 +9,10 @@ use crate::{
 /// Provider of runtime host functionality.
 pub trait RuntimeProvider {
     /// This method should return the caller of the current context.
-    fn get_caller(&self) -> AccountHash;
+    fn get_caller(&self) -> Result<AccountHash, Error>;
```

Comment:
Thats still TBD but in case https://casperlabs.atlassian.net/browse/EE-1125 requires that we'd charge for host functions called inside system contracts it would be necessary to have a Result here (for gaslimit etc)

Reply:
I'd prefer that we only add the `Result` when we actually make such a change.

Reply:
Fair point - removed in a48109152118b148d0b3f5626504bc5912623131